src/tests/StreamerTest.cpp

```diff
@@ -228,6 +239,37 @@ class StreamerProcessTimingTest : public ::testing::Test {
   StreamerStandIn TestStreamer;
 };
 
+TEST_F(StreamerProcessTimingTest, MessageHasNoTimestamp) {
+  FlatbufferReaderRegistry::Registrar<StreamerNoTimestampTestDummyReader>
```

Comment:
Could that throw if schema id is already registered? Tests are run in undefined order in general.

Reply:
Really don't think so (and I've never seen it throwing)
- the reader is registered only there
- other reader are registered multiple times in different units